### PR TITLE
Apply notification settings.

### PR DESCRIPTION
**Pull request by aforge**
Actually use UiUtils.PREF_READ_RCPT and UiUtils.PREF_TYPING_NOTIF
in MessageActivity.

### CODE REVIEW

File: app/src/main/java/co/tinode/tindroid/MessageActivity.java [hunk 1]
```diff
@@ -202,6 +209,14 @@ public void onResume() {
                 showFragment(FRAGMENT_FILE_PREVIEW, args, true);
             }
         }
+
+        final SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(this);
+
+        mSendReadReceipts = pref.getBoolean(UiUtils.PREF_READ_RCPT, true);
+        Log.i(TAG, "Topic '" + topicName + "': sendReadReceipts = " + mSendReadReceipts);
```
Comment (or-else) [inline app/src/main/java/co/tinode/tindroid/MessageActivity.java:216]: This looks like debugging output and completely unnecessary once debugged.

File: app/src/main/java/co/tinode/tindroid/MessageActivity.java [hunk 2]
```diff
@@ -202,6 +209,14 @@ public void onResume() {
...
+        mSendTypingNotifications = pref.getBoolean(UiUtils.PREF_TYPING_NOTIF, true);
+        Log.w(TAG, "Topic '" + topicName + "': sendTypingNotifications = " +
```
Comment (or-else) [inline app/src/main/java/co/tinode/tindroid/MessageActivity.java:218]: Same as above.

File: app/src/main/java/co/tinode/tindroid/MessageActivity.java (hunk 1 above)
Comment (aforge) [reply app/src/main/java/co/tinode/tindroid/MessageActivity.java]: Indeed. Removed.

File: app/src/main/java/co/tinode/tindroid/MessageActivity.java (hunk 2 above)
Comment (aforge) [reply app/src/main/java/co/tinode/tindroid/MessageActivity.java]: Done.

**Pull request merged**